staging/src/k8s.io/apiserver/pkg/storage/storagebackend/factory/etcd3.go

```diff
@@ -27,6 +29,11 @@ import (
 	"k8s.io/apiserver/pkg/storage/value"
 )
 
+var (
+	keepaliveTime    = 30 * time.Second
```

Comment:
add comments?

Reply:
keepalive intervals and timeouts are pretty well known and the variables are not exported. Is there something that could be cleared up with a comment?

Reply:
@rphillips we might want to document why we choose a aggressive timeout (there will be only a couple of api servers connected to etcd server as clients. an aggressive timeout can help to detect failure faster without introducing much overheads)

Reply:
added a comment